base_exception/security/base_exception_security.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```

Comment:
add copyright

Reply:
copyright is not needed, but you can view an example on this link
https://github.com/OCA/server-tools/blob/12.0/module_auto_update/views/ir_module_module.xml#L4